rdm/0002-data-models-terminology.md

```diff
@@ -0,0 +1,38 @@
+- Start Date: 2019-09-30
+- RFC PR: [#2](https://github.com/inveniosoftware/rfcs/pull/2)
+- Authors: Lars Holm Nielsen
+
+# Data models terminology
+
+## Summary
+
+Definition of terms used when speaking about records and data models.
+
+## Motivation
+
+The purpose of this RFC is to disambiguate terms like records, record type, schema and data model that are often used interchangeably in the context of Invenio.
+
+## Detailed design
+
+The central part of InvenioRDM is the primary data model for storing a bibliographic record type describing the digital resources (datasets, software, articles, ...). In addition to the primary data model, InvenioRDM can be extended with one or more secondary data models for storing authority record types (grants, licenses, ...).
```

Comment:
@fenekku you are right it is more accurate like that. 
On the other hand, I am not sure if I would add the custom data model fields here; just to keep the already complex explanations simple. The main message is, there 2 data models: 1 primary (bibliographic record) and >=0 secondary data models (authorities).  Then, whether they are extensible or not is a different business :) What do you think?

Reply:
I do want to mention the "data model extension" because the RFC tries to lay out the terms we will be using and we want a term to refer to that concept of extending/customizing the primary data model. How about putting it in a different paragraph like so:

 ```suggestion
The central part of InvenioRDM is the primary data model for storing a bibliographic record type describing the digital resources (datasets, software, articles, ...). In addition, InvenioRDM supports one or more separate secondary data models for storing authority record types (grants, licenses, ...).

The primary data model can be extended with flavour-specific fields that we refer to as data model extensions.
```

?

Reply:
It looks better! We use to say the word "flavour" to describe different apps built with Invenio, for instance: RDM, ILS, Multimedia Archive. It is not the case now I think so if everybody agrees to use it here as well, no problem with me. Otherwise we could simply say "custom fields", "per-case custom fields" , or something similar.

Reply:
Ah, we have the flavour / profile RFC [here](https://github.com/inveniosoftware/rfcs/pull/6) for you kind sir! So "flavour" here has a specific meaning.